logsearch-deployment.yml

```diff
@@ -1,104 +1,100 @@
----
 meta:
-  <<: (( merge ))
-  environment: ~
-  releases:
-  - name: logsearch
-    version: latest
-  - name: logsearch-for-cloudfoundry
-    version: latest
-  - name: fisma
-    version: latest
-  - name: collectd
-    version: latest
-  - name: tripwire
-    version: latest
-  - name: awslogs
-    version: latest
-  - name: nessus-agent
-    version: latest
-  - name: newrelic
-    version: latest
-  - name: riemannc
-    version: latest
-  - name: riemann
-    version: latest
-  - name: clamav
-    version: latest
-  - name: snort
-    version: latest
   default_env:
     # Default vcap & root password on deployed VMs (ie c1oudc0w)
     # Generated using mkpasswd -m sha-512
     bosh:
-      password: (( merge || "$6$4gDD3aV0rdqlrKC$2axHCxGKIObs6tAmMTqYCspcdvQXh3JJcvWOY2WGb4SrdXtnCyNaWlrf3WEqvYR2MYizEGp3kMmbpwBC6jsHt0" ))
-  stemcell: (( merge ))
-  disk_size:
-    elasticsearch_master: 50_000
-    elasticsearch_data: 500_000
-    queue: 75_000
+      password: "$6$4gDD3aV0rdqlrKC$2axHCxGKIObs6tAmMTqYCspcdvQXh3JJcvWOY2WGb4SrdXtnCyNaWlrf3WEqvYR2MYizEGp3kMmbpwBC6jsHt0"
```

Comment:
I restored the usual `password: (( merge || "..." ))`, but spruce doesn't allow it--and the password will be merged from secrets anyway. If we want to ensure that the default password is never used, we could use `password: ~` instead.